app/views/content/life-as-a-teacher/pay-and-benefits/teacher-pay.md

```diff
@@ -92,7 +92,7 @@ Some schools also have a leading practitioner pay range for teachers with exempl
 What this will involve will depend on your school, but could include:
 
 * coaching and mentoring other teachers, including trainees and early career teachers
-* undertaking a national professional qualification (NPQ)
+* undertaking a national professional qualification (NPQ), such as the SENCO NPQ to [specialise in teaching children with special educational needs](/life-as-a-teacher/pay-and-benefits/teacher-pay)
```

Comment:
Suggest spelling out SENCO in full as think it's the first time it's mentioned on this page?